### ISSUE TITLE
 do not know port number used by SSH build in server

### ISSUE THREAD

**Issue by JanuszChmiel** [CypherpunkArmory/UserLAnd#152]
Dear specialists,
Since I do not see at all, I can not use Connect bot to browse terminal. But I can not gues Port number myself. I want to use Putty for Windows to connect from laptop with Windows and NVDA screen reader to read console output here. But since I do not know The port number, I can not connect by using Putty for Windows. Port 2222 or 22 do not work from putty.
Or app have been designed to work only with Connect bot so it will not allow other incoming SSH requests than those, which have been established by Connect bot?
Talkback or other screen readers traditionally do not support various Linux consoles so also yours app is not The exception. This is The reason, why I Am using this approach.
Any help will be very welcomed.
With kindness regards.
Janusz Chmiel

**Comment by bokuno** [comment]
The connectBot says:
port:2022,
host:localhost

**Comment by corbinlc** [comment]
That is correct.  It is 2022.  You should be able to connect from another device on the same network.

**Comment by JanuszChmiel** [comment]
I have good news for all of all.
Connect Bot is perfectly accessible, but gesture of Talkback which read automatically object by object when swiping from left to right or from right to left did not announced a long text field which contain all text output. So I can perfectly use yours project even on The Android device with no need to relay on other computer. Talkback is strane and in some cases I must explore by touch randomly by moving finger across The display area, I can not always relay on simple build in gesture. So I AM very sorry that I was intruded you so much. By The way, all subsystems are very stable, no crashes. I will test XSDL with Gnome or Mate. I will also try to grab hackers keyboard from Google Play to test if Talkback will announce keypresses for Me when inside Connect bot terminal window.
